src/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -0,0 +1,245 @@
+          href={item.href || '#'}
+          style={{
+            cursor: 'pointer',
+            textDecoration: 'none',
+            color: '#000',
+            fontWeight: 'bold',
+            fontSize: isMobileView ? 16 : 20,
+            width: '100%',
+            textAlign: 'center',
+            padding: '8px 0',
+            borderRadius: '8px',
+            backgroundColor: '#C2F9F9',
+          }}
+        >
+          {item.label}
+        </a>
+      ))}
+    </Box>
+  );
+
+  return (
+    <DashboardBackground>
+      <NoiseLayer src="/image.png" alt="noise texture" />
+      <MaskLayer />
+      <Line15 />
+
+      <ResponsiveContainer isMobile={isMobile}>
+        <Box mb={3} className="logo">
+          <Logo orientation="vertical" width={isMobile ? 71 : 192} height={isMobile ? 56 : 88} />
+        </Box>
+
+        {!isMobile && (
+          <Frame71Placeholder isMobile={isMobile}>
+            {renderNavigationLinks(false)}
+          </Frame71Placeholder>
+        )}
+
+        {isMobile && (
+          <MobileMenuButton
+            isMobile={isMobile}
+            onClick={() => setDrawerOpen(true)}
+            aria-label="Open Dashboard Navigation"
+          >
+            <img src={menuIcon} alt="Menu" width={36} height={36} />
+          </MobileMenuButton>
+        )}
+
+        {isMobile && (
+          <MobileDrawer anchor="right" open={drawerOpen} onClose={() => setDrawerOpen(false)}>
+            <Box p={2}>
+              <h3>Dashboard Navigation</h3>
+              {renderNavigationLinks(true)}
+              <button onClick={() => setDrawerOpen(false)}>Close Menu</button>
+            </Box>
+          </MobileDrawer>
+        )}
+
+        <Group59Container>
+          <Group13 isMobile={isMobile}>
+            <WelcomeMessage>Welcome {user.name}!</WelcomeMessage>
+            <ProfileSubtitle>Here is your Profile Dashboard</ProfileSubtitle>
+          </Group13>
+
+          {!isMobile && (
+            <Box position="relative">
+              <Group49Placeholder
+                src={Group49Img}
+                alt="notification"
+                onClick={() => setNotificationOpen((prev) => !prev)}
+                style={{ cursor: 'pointer' }}
+              />
+              {notificationOpen && (
+                <Group16>
+                  <NotificationText>You have 12 new action items!</NotificationText>
+                </Group16>
+              )}
+            </Box>
+          )}
+
+          <Group53Placeholder isMobile={isMobile}>
+            <SearchBar placeholder="Search..." debounceDelay={300} autoSearch />
+          </Group53Placeholder>
+
+          <AvatarContainer isMobile={isMobile}>
+            <AvatarMenu
+              avatarUrl="https://i.ibb.co/mV26g4B7/b2cc1d5a59644f92c2391dcdd5cde3c11e4770fe.jpg"
+              username="Miguel Maquera"
+              menuItems={[
+                { label: 'Profile', onClick: () => (window.location.href = '/profile') },
```

Comment:
Menu items use imperative navigation via window.location.href. This couples the component to browser navigation and makes unit testing and SPA routing harder.

**Sugerencia:**
```typescript
Use a navigation abstraction (e.g., pass onClick handlers from parent, use react-router's navigate, or a centralized navigation function) so tests can assert navigation without changing window.location and to support SPA routing.
```